.github/workflows/python_safety.yml

```
@@ -8,9 +8,12 @@ jobs:
     runs-on: 'ubuntu-20.04'
     steps:
       - run: sudo apt-get -q update
-      - run: sudo DEBIAN_FRONTEND="noninteractive" apt-get -qq --no-install-recommends dist-upgrade
       - run: sudo DEBIAN_FRONTEND="noninteractive" apt-get -qq --no-install-recommends install
              curl gcc ffmpeg libcurl4-openssl-dev libssl-dev motion v4l-utils
       - uses: actions/checkout@v4
-      - run: pip install --user safety .
+      - uses: actions/setup-python@v4
+        with:
+          python-version: 3.x
+      - run: pip install --upgrade pip setuptools      
```

Probably with the `-U` at least all dependencies are installed with max possible versions?
```suggestion
      - run: pip install --upgrade pip setuptools
      - run: pip install -U .
```